cron/syncAlertsToMongo.py

```diff
@@ -74,15 +52,15 @@ def updateMongo(mozdefdb, esAlerts):
             mrecord = a['_source']
             # generate a meteor-compatible ID
             mrecord['_id'] = genMeteorID()
-            # capture the elastic search meta data (index/id/doctype)
+            # capture the elastic search meta data (index/id/type)
             # set the date back to a datetime from unicode, so mongo/meteor can properly sort, select.
             mrecord['utctimestamp']=toUTC(mrecord['utctimestamp'])
             # also set an epoch time field so minimongo can sort
             mrecord['utcepoch'] = calendar.timegm(mrecord['utctimestamp'].utctimetuple())
             mrecord['esmetadata'] = dict()
             mrecord['esmetadata']['id'] = a['_id']
             mrecord['esmetadata']['index'] = a['_index']
-            mrecord['esmetadata']['type'] = a['_type']
+            mrecord['esmetadata']['type'] = a['type']
```

Comment:
This isn't the correct location of `type`, it's in `a['_source']['type']

Reply:
I think we can actually delete this line entirely, as the type of the event is no longer esmetadata.